src/AutoMapper/Bundle/Configuration/LegacyConfigurationPassDecorator.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Jane\AutoMapper\Bundle\Configuration;
+
+use Jane\AutoMapper\MapperGeneratorMetadataInterface;
+
+/**
+ * @deprecated since 6.3, will be removed in 7.0. Please use `Jane\AutoMapper\Bundle\Configuration\MapperConfigurationInterface` instead.
+ */
```

Comment:
I added the same comment on this class as it's here only to avoid BC breaks and will be removed as well in 7.0